src/System.Numerics.Tensors/pkg/System.Numerics.Tensors.pkgproj

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <PropertyGroup>
+    <!-- we need to be supported on pre-nuget-3 platforms (Dev12, Dev11, etc) -->
+    <MinClientVersion>2.8.6</MinClientVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Numerics.Tensors.csproj" />
```

Comment:
We need to annotate this with SupportedFrameworks to indicate the frameworks we expect this package to support.   See https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/package-projects.md#simple-portable-library

Reply:
Is this needed for both `ref` and `src` or just `ref`?

Reply:
Just Ref

---
In reply to: [231687922](https://github.com/dotnet/corefx/pull/33314#discussion_r231687922) [](ancestors = 231687922)